src/SDK/SmartDetectors.ProjectType/app.config

```diff
@@ -12,7 +12,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0" />
```

Comment:
>1 [](start = 49, length = 1)

I think we are using this version in SDRE on purpose since we had dll mismatch.
why do you need to upgrade this?

Reply:
had build warning on assembly mismatch
in any case - it doesn't affect the SDRE since it's not part of the SDK

---
In reply to: [258436742](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/50#discussion_r258436742) [](ancestors = 258436742)